packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -0,0 +1,243 @@
+   */
+  accessibilityControls?: string;
+  /**
+   * Label for screen readers to announce ButtonToggle. See the [Accessibility guidelines](https://gestalt.pinterest.systems/web/buttontoggle#ARIA-attributes) for details on proper usage.
+   */
+  accessibilityLabel?: string;
+  /**
+   * The background color of ButtonToggle.
+   */
+  color?: 'red' | 'transparent';
+  /**
+   * Available for testing purposes, if needed. Consider [better queries](https://testing-library.com/docs/queries/afut/#priority) before using this prop.
+   */
+  dataTestId?: string;
+  /**
+   * Indicates if ButtonToggle is disabled. Disabled ButtonToggles are inactive and cannot be interacted with. See the [state variant](https://gestalt.pinterest.systems/web/buttontoggle#State) for details on proper usage.
+   */
+  disabled?: boolean;
+  /**
+   * An icon displayed before the text to help clarify the usage of ButtonToggle.
+   */
+  iconStart?: keyof typeof icons;
+  /**
+   * Callback invoked when the user clicks (press and release) on ButtonToggle with the mouse or keyboard.
+   */
+  onClick?: (arg1: {
+    event: React.MouseEvent<HTMLButtonElement> | React.KeyboardEvent<HTMLButtonElement>;
+  }) => void;
+  /**
+   * Toggles between selected/unselected.
+   */
+  selected?: boolean;
+  /**
+   * sm: 32px, md: 40px, lg: 48px
+   *
+   * See the [size variant](https://gestalt.pinterest.systems/web/buttontoggle#Size) variant to learn more.
+   */
+  size?: 'sm' | 'md' | 'lg';
+  /**
+   * Text to render inside the ButtonToggle to convey the function and purpose of the ButtonToggle.
+   */
+  text: string;
+};
+
+function InternalButtonContent({
+  target,
+  text,
+  textColor,
+  icon,
+  size,
+}: {
+  target?: Target;
+  text: ReactNode;
+  textColor: IconColor;
+  icon?: keyof typeof icons;
+  size: string;
+}) {
+  return (
+    <Fragment>
+      <Flex alignItems="center" gap={{ row: 2, column: 0 }} justifyContent="center">
+        {icon ? (
+          <Icon
+            accessibilityLabel=""
+            color={textColor}
+            icon={icon}
+            // @ts-expect-error - TS7053 - Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ readonly sm: 10; readonly md: 12; readonly lg: 12; }'.
+            size={SIZE_NAME_TO_PIXEL[size]}
+          />
+        ) : null}
+        {text}
+      </Flex>
+      <NewTabAccessibilityLabel target={target} />
+    </Fragment>
+  );
+}
+
+/**
+ * [ButtonToggles](https://gestalt.pinterest.systems/web/buttontoggle) is a larger alternative to selection components such as [Checkbox](/android/checkbox), [RadioButton](/web/radiobutton), and [Switch](/android/switch). It enables users to choose between two states - selected or unselected.
```

Comment:
ButtonToggle

Reply:
we need full links /web/radiobutton cos its supposed to go to VSCode extension too

Reply:
Fixed!